Documentation/EyeTracking/EyeTracking_BasicSetup.md

```diff
@@ -7,9 +7,11 @@ Alternatively, you can check out our already configured [MRTK Eye Tracking Examp
 For Eye Tracking to work correctly, the following requirements must be met:
 
 1. An _'Eye Gaze Data Provider'_ must be added to the input system. This provides eye tracking data from the platform.
-2. Eye Tracking must be set as the preferred source in the _'Gaze Provider'_. This setting can be toggled at runtime.
-3. The _'Gaze Input'_ capability must be enabled in the application manifest. **Currently this is only available in Visual Studio.**
-4. The HoloLens **must** be calibrated for the current user under system settings.
+2. The _'Gaze Input'_ capability must be enabled in the application manifest. **Currently this is only available in Visual Studio.**
+3. The HoloLens **must** be calibrated for the current user under system settings.
+4. The GazeProvider must have its "Use Eye Tracking" property set to true. Note that true is the default value (so no special
```

Comment:
Hm, the order is a bit weird. Maybe we should address the initial things to check in the Unity Editor first, then VS, then the generic hint that the user has to be calibrated. By the way, statement number 3 sounds like that you must calibrate via System Settings, which is incorrect. The calibration should be automatically triggered once a new user puts on the device. Only in case the user skipped that step for some reason, they would need to call it up via System Settings.

Reply:
Fair point, switched back to original order.

I had done things this way because it was the least important thing for people to be aware of (i.e. it's a no-op) so having it in the middle actually causes them to have to spend attention on something that isn't actually important.

Reply:
Yeah, I understood where you're coming from. We could have two sections: 1. Assuming you start out with our default setup. 2. "Troubleshoot" section: "What to check if your Eye Targeting isn't working?"